staging/src/k8s.io/externaljwt/apis/v1alpha1/api.pb.go

```diff
@@ -0,0 +1,590 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: api.proto
+
+package v1alpha1
+
+import (
+	context "context"
+	fmt "fmt"
+	proto "github.com/gogo/protobuf/proto"
+	grpc "google.golang.org/grpc"
+	codes "google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	math "math"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+type SignJWTRequest struct {
+	// URL-safe base64 wrapped payload to be signed.
+	// Exactly as it appears in the second segment of the JWT
+	Claims               string   `protobuf:"bytes,1,opt,name=claims,proto3" json:"claims,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *SignJWTRequest) Reset()         { *m = SignJWTRequest{} }
+func (m *SignJWTRequest) String() string { return proto.CompactTextString(m) }
+func (*SignJWTRequest) ProtoMessage()    {}
+func (*SignJWTRequest) Descriptor() ([]byte, []int) {
+	return fileDescriptor_00212fb1f9d3bf1c, []int{0}
+}
+func (m *SignJWTRequest) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_SignJWTRequest.Unmarshal(m, b)
+}
+func (m *SignJWTRequest) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_SignJWTRequest.Marshal(b, m, deterministic)
+}
+func (m *SignJWTRequest) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_SignJWTRequest.Merge(m, src)
+}
+func (m *SignJWTRequest) XXX_Size() int {
+	return xxx_messageInfo_SignJWTRequest.Size(m)
+}
+func (m *SignJWTRequest) XXX_DiscardUnknown() {
+	xxx_messageInfo_SignJWTRequest.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_SignJWTRequest proto.InternalMessageInfo
+
+func (m *SignJWTRequest) GetClaims() string {
+	if m != nil {
+		return m.Claims
+	}
+	return ""
+}
+
+type SignJWTResponse struct {
+	// header must contain only alg, kid, typ claims.
+	// type must be “JWT”.
```

Comment:
this has drifted from the proto file, need to run `hack/update-codegen.sh proto`

Reply:
Regenerated